autotools: --enable-wayland broken when using dash as CONFIG_SHELL

Since commit 78431a361bed0b6e9f1f3ebc0de8897b069b3848 using autotools with GNU sed to build libva fails when --enable-wayland is set:
```
Making all in wayland
make[3]: Entering directory '/var/tmp/portage/x11-libs/libva-2.10.0/work/libva-2.10.0-abi_x86_64.amd64/va/wayland'
/usr/bin/wayland-scanner client-header < /var/tmp/portage/x11-libs/libva-2.10.0/work/libva-2.10.0/va/wayland/wayland-drm.xml > wayland-drm-client-protocol.h
/usr/bin/wayland-scanner code < /var/tmp/portage/x11-libs/libva-2.10.0/work/libva-2.10.0/va/wayland/wayland-drm.xml > wayland-drm-client-protocol-export.c
Using "code" is deprecated - use private-code or public-code.
See the help page for details.
/bin/sed -e $'1i\\\n#include "sysdeps.h"\n' \
                   -e 's@WL_EXPORT@DLL_HIDDEN@g' < wayland-drm-client-protocol-export.c > wayland-drm-client-protocol.c
/bin/sed: -e expression #1, char 2: unknown command: `1'
make[3]: *** [Makefile:768: wayland-drm-client-protocol.c] Error 1
```

My sed is:
```
# sed --version | head -n1
sed (GNU sed) 4.8
```
Reverting commit 78431a361bed0b6e9f1f3ebc0de8897b069b3848 makes the build succeed.

--- a/va/wayland/Makefile.am
+++ b/va/wayland/Makefile.am
@@ -70,8 +70,9 @@ va_wayland_drm.c: $(protocol_source_h)
 %-client-protocol-export.c : %.xml
 	$(AM_V_GEN)$(WAYLAND_SCANNER) code < $< > $@
 %-client-protocol.c: %-client-protocol-export.c
-	$(AM_V_GEN)$(SED) -e $$'1i\\\n#include "sysdeps.h"\n' \
-	                   -e 's@WL_EXPORT@DLL_HIDDEN@g' < $< > $@
+	$(AM_V_GEN){ echo '#include "sysdeps.h"'; $(SED) \
+		-e 's@WL_EXPORT@DLL_HIDDEN@g' \
+		< $<; } > $@
 
 EXTRA_DIST = \
 	wayland-drm.xml         \

